welcome/setup.html

```diff
@@ -0,0 +1,34 @@
+<ol class="experimental-platform-setup">
+  <li>
+    <h4>Install Vagrant and VirtualBox</h4>
```

Comment:
yeah, that was on purpose (I know I had it differently before too) - the idea being that at first you will go the "virtual way" and won't have the hardware ready. This first page is developer centric, with another one happening for "makers" at some point later